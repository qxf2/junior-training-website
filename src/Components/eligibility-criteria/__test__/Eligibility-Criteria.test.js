import React from 'react';
import ReactDOM from 'react-dom';
import Eligibility from '../Eligibility-Criteria';

import { render, cleanup } from '@testing-library/react';
import "@testing-library/jest-dom/extend-expect";

import renderer from "react-test-renderer";

afterEach(cleanup);

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Eligibility/>, div);
});

it("renders Eligibility correctly", () => {
   const { getByTestId }  = render(<Eligibility/>)
   expect(getByTestId('eligibility'));
});

it("Eligibility-criteria.js matches snapshot", () =>{
  const tree = renderer.create(<Eligibility/>).toJSON();
  expect(tree).toMatchSnapshot();
});