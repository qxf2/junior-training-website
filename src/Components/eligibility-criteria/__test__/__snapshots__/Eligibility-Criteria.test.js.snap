// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Eligibility-criteria.js matches snapshot 1`] = `
<section
  data-testid="eligibility"
  id="eligibility"
>
  <div
    className="react-reveal row"
    style={
      Object {
        "opacity": undefined,
      }
    }
  >
    <h4
      className="react-reveal"
      style={
        Object {
          "opacity": undefined,
        }
      }
    >
      ELIGIBILITY CRITERIA
    </h4>
    <div
      className="twelve columns collapsed"
    >
      <div
        className="bgrid-quarters s-bgrid-thirds cf"
      >
        <ol>
          <li>
            Quick learner
          </li>
          <p>
            You should be comfortable with a fast pace. Since we do not provide classroom training, our pace can seem fast for most trainees.
          </p>
          <li>
            Self starter
          </li>
          <p>
            Doing a project on your own interest is a good indicator that you will be able to complete the training program.
          </p>
          <li>
            Demoable work
          </li>
           
          <p>
            You should have a GitHub profile or a project video for hardware projects.
          </p>
          <li>
            Recent graduate
          </li>
           
          <p>
            This program is available only to 2020 and 2021 graduates.
          </p>
        </ol>
      </div>
    </div>
  </div>
</section>
`;
